tests/config.tests/mapping.json

```diff
@@ -0,0 +1,29 @@
+//
+// This message represents the entire high-level config of a mapping recommendation model.
+// This is essentially an "input" from the outside world that might be relevant to the mapper internals.
+//
+// The building id is included as part of the message envelope.
+//
+// Basic comparisons are with values in the related state message.json block, which can be handled as logical
+// timestamp comparisons:
+//    applied < promoted: This means a entry has been promoted but has not yet been applied by the external system
+//    requested > exported: This means that an export has been requested but not yet triggered (mapper should export data)
+//
+{
+  "version": "1.3.14",
+  "timestamp": "2018-08-28T21:39:29.364Z",
+  "mapping": {
+    "devices": {
+      "FCU-123": {
+        "applied": "2018-08-27T22:39:12.364Z", // Last time a promotion was successfully applied externally
+        "requested": "2018-08-27T22:39:12.364Z", // Timestamp of requested model export
+        "status": {
+          "message": "Update in progress",
+          "category": "mapping.apply.start",
+          "timestamp": "2018-08-28T21:39:30.364Z",
+          "level": 500
+        }
+      }
```

Comment:
The mapping must contain the following:

Which building is it for?
the cloud device id for the device.
Any dependencies such as links to another entity or connection to another entity must be present as well.


```
6798-guid:
  code: CHWS_WDT-17
  type: HVAC/CHWS_WDT
  cloud_device_id: 123456
  guid: CHWS_WDT-17-guid
  translation:
    return_water_temperature_sensor:
      present_value: "points.return_water_temperature_sensor.present_value"
      units:
        key: "pointset.return_water_temperature_sensor.units"
        values:
          degrees_celsius: "degC"
    supply_water_temperature_sensor:
      present_value: "points.supply_water_temperature_sensor.present_value"
      units:
        key: "pointset.points.supply_water_temperature_sensor.units"
        values:
          degrees_celsius: "degC"

1234-guid:
  type: FACILITIES/BUILDING
  code: UK-LON-S2
  ```

Reply:
The actually mapping info is in the other file/message -- that's in tests/event_mapping.tests/building_config.json  -- I'm unopinionated about the semantics contents of that message, so I could just take the snippet you had here and put it there. I don't understand the structure of the message, but that doesn't really matter.

Reply:
yes my bad, it is the mapping info